apps/aecontract/test/aecontract_SUITE.erl

```diff
@@ -3466,42 +3483,19 @@ merge_new_zero_value(_Cfg) ->
     Store1 = #{ <<0>> => <<42>>
               , <<1>> => <<0>>
               , <<2>> => <<>> },
-    Ct2    = aect_contracts:set_state(Store1, Ct1),
+    Ct2    = set_ct_store(Store1, Ct1),
     Ct2    = ?call(enter_contract, Ct2),
     %% Empty values are removed in state tree.
     Ct3    = ?call(get_contract, Ct2),
     Store2 = #{ <<0>> => <<0>>
               , <<1>> => <<>>
               , <<2>> => <<42>> },
-    Ct4    = aect_contracts:set_state(Store2, Ct3),
+    Ct4    = set_ct_store(Store2, Ct3),
     Ct4    = ?call(enter_contract, Ct4),
     Ct5    = ?call(get_contract, Ct4),
     Store3 = #{ <<0>> => <<0>>
               , <<2>> => <<42>>},
-    Store3 = aect_contracts:state(Ct5),
-    ok.
-
-
-merge_missing_keys(_Cfg) ->
```

Comment:
Why is this test removed?

Reply:
In the new representation you remove things by writing `<<>>` so there is no merging of missing keys. If a key is not in the write cache it means it doesn't change.